### PR TITLE
Add index.d.ts

### PR DESCRIPTION
**Pull request by VincentLanglet**
I started to work on an index.d.ts (cf https://github.com/Instabug/instabug-reactnative/issues/153)

It's not perfect, but can do the job. I am open to advice !

### CODE REVIEW

**Comment by salmatarzi** [comment]
@VincentLanglet Thank you for this 🙏🏽We will be reviewing and testing it, then we'll get back to you!

**Comment by VincentLanglet** [comment]
@salmatarzi Thanks, did you have time to review ?

**Comment by VincentLanglet** [comment]
@salmatarzi Up :)

**Comment by VincentLanglet** [comment]
@salmatarzi Do you plan to re-read this PR or is it easier if I do it on DefinitivelyTyped ?

**Comment by VincentLanglet** [comment]
@salmatarzi Up :)

**Comment by HeshamMegid** [comment]
Sorry for the delay, this fell off our radar. We're reviewing/testing this and planning to include it into our next release in a couple of weeks if everything looks fine.

**Comment by VincentLanglet** [comment]
Thanks

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "2.11.2",
   "description": "React Native plugin for integrating the Instabug SDK",
   "main": "index.js",
+  "type": "index.d.ts",
```
Comment (VincentLanglet) [inline package.json]: Should be "types"

**Pull request closed without merging**